Sources/Queryable/Handlers/Queryable+Conditional.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  IfQueryable.swift
+//
+//
+//  Created by Kai Quan Tay on 12/1/24.
+//
+
+import SwiftUI
+
+public struct IfQueryable<Item, Result, QueryContent: View>: View {
```

Comment:
I'd prefer the name `QueryableView`. `IfQueryable` doesn't read so nicely, imho.

```suggestion
public struct QueryableView<Item, Result, QueryContent: View>: View {
```

Reply:
I think that `QueryableView` doesn't get the point across that it conditionally shows if a query is active. 

`IfQuery` reads a bit better, and gets the point across that its a conditional.

Reply:
Yeah, I get your point. My thinking was that `QueryableView` tells the user it is "a view that can be queried", but I see how that might not be everybody's first intuition when seeing the name 🤔 

I am okay with calling it `WithQuery` 👍